conference-demos/onnxruntime-openvino-lcm-demo/Latent Consistency Models with ONNX and OpenVINO Execution Provider.ipynb

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
<p>Isn't better to use just GPU instead of GPU.0? This bug will be finally fixed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2002/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Let's keep it GPU.0, for now, to keep notebook runnable and fix later alone with "GPU.0" fix